src/commands/publish-local.js

```diff
@@ -78,29 +77,25 @@ export function publishLocal(config = { config: { specs: [] } }) {
 
             artifactIds.push(paramCase(api.info.title));
 
-            // Publish!!
-            // copy archive file into config.repoPath / config.groupId / paramCase(api.info.title) / api.info.version
-            const target = `${config.repoPath}/${config.groupId.replace(
-              /\./g,
-              '/'
-            )}/${paramCase(api.info.title)}/${api.info.version}`;
-            if (!fs.existsSync(target) || !fs.lstatSync(target).isDirectory()) {
-              mkdirp.sync(target);
-            }
-
-            // Copy zip
-            const artifactName = path.parse(archive).name;
-            fs.copyFileSync(archive, `${target}/${artifactName}.zip`);
-
-            // Copy POM
+            // Write temporary pom file
             const pomContent = getPOMContent(
               paramCase(api.info.title),
               api.info.version,
               config.groupId,
               'zip'
             );
 
-            fs.writeFileSync(`${target}/${artifactName}.pom`, pomContent);
+            const pomFile = `${getTempDir().name}/${
+              path.parse(archive).name
+            }.pom`;
+            fs.writeFileSync(pomFile, pomContent);
+
+            // Install with maven plugin
+            const mvn = require('maven').create({ quiet: true });
+            await mvn.execute(['install:install-file'], {
```

Comment:
With your code, is seems that `--repoPath` parameter is no longer used. You should change README, config-validation and config-definition files